Correct artifact stanza in google-earth-web-plugin

winmillwill opened the pull request:
It seem that the nuance of creating a plugin file in `/Library/Internet\ Plug-ins` is handled by the maintainers with logic they rolled into a pkg.

Despite the name of the .pkg file, it seems to have just installed the plugin and not the full Google Earth app, so it seems that this cask is still valid. Without this change, installation fails.

vitorgalvao commented:
It’s missing `uninstall`. There are [scripts](https://github.com/caskroom/homebrew-cask/tree/master/developer/bin) and [documentation](https://github.com/caskroom/homebrew-cask/blob/master/doc/CASK_LANGUAGE_REFERENCE.md#uninstall-stanza-details) geared for that. Do you need any help?

winmillwill commented:
Added the uninstall and squashed.

There is some mention of this other pkg id: `com.google.pkg.Keystone`, which has no associated files and seems to be related to the updater tool for Google products. I don't know what the consequences would be for adding this to the pkgutil uninstall list, and I would worry that it would affect other applications, like Chrome itself.

vitorgalvao commented:
Agreed.
